src/modules/form.js

```diff
@@ -0,0 +1,73 @@
+// Add these lines at the top of the file
+import { todoLists, displayTodoList } from './todos.js';
+
+const form = document.getElementById('todoForm');
+const todoInput = document.getElementById('newtodo');
+const errorMessag = document.getElementById('errorMessag');
+const todosListParent = document.getElementById('todo-list');
+
+// Define the removeTodo function
+// eslint-disable-next-line no-use-before-define
+const removeTodo = (index) => { // eslint-disable-line
```

Comment:
- [ ] When removing a task from the list the index should replace the old removed task, kindly consider fixing this so all the indexes are displayed correctly

![Screenshot_20230623_000204](https://github.com/akbarsazish/To-Do-List/assets/25227022/86b9e7e1-560d-4ba1-99a7-d689f1a1c79e)